packages/mock-block-dock/dev/dev-app.tsx

```diff
@@ -88,6 +95,7 @@ const DevApp: FunctionComponent = () => {
         debug
         blockDefinition={blockDefinition}
         blockEntity={blockEntity}
+        blockName="Test Block"
```

Comment:
There's a new `blockName` prop which is displayed at the top of the debug UI. This property will be pulled from `blockprotocol.displayName` field in the block's package.json file. See [internal link](https://www.figma.com/file/Fpmde3E9nJMb4VLkPTYgrM?node-id=3896:28985#252029946)

Reply:
@CiaranMn what do you think about adding a `blockInfo` prop which is the content pulled from "blockprotocol" key in the block's package.json?

Reply:
Yes - I think we should do that in our block templates

Reply:
Updated MockBlockDock to accept a blockInfo prop in https://github.com/blockprotocol/blockprotocol/pull/555. Will update block templates in  #555